react-form-controls/src/registration-form.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default class RegistrationForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: ''
+    };
+    this.handleUsernameChange = this.handleUsernameChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleUsernameChange(event) {
+    this.setState({ username: event.target.value });
+  }
+
+  handlePasswordChange(event) {
+    this.setState({ password: event.target.value });
+  }
+
+  handleSubmit(event) {
+    console.log('state: ', this.state);
+    event.preventDefault();
+  }
+
+  render() {
+    return (
+      <form onSubmit={ this.handleSubmit }>
+        <label htmlFor="username">Username:</label>
+        <input type="text" name="username" placeholder="JonSnow" value={ this.state.username } onChange={ this.handleUsernameChange } />
```

Comment:
You know nothing Jon Snow

Reply:
![image](https://user-images.githubusercontent.com/76730244/189808112-38433127-dc59-4c71-aaa1-f0596bf0ad7b.png)